packages/docusaurus-theme-classic/src/theme/SkipToContent/index.tsx

```diff
@@ -19,6 +20,7 @@ function programmaticFocus(el: HTMLElement) {
 
 function SkipToContent(): JSX.Element {
   const containerRef = useRef<HTMLDivElement>(null);
+  const {action} = useHistory();
```

Comment:
Perhaps `action.history` should be part of `location` object in `useLocationChange` hook?

Reply:
Not really needed until we have a 2nd use-case to access history.action so as you want. Will merge it in current state for now